src/highlight/__tests__/HighlightListener-test.ts

```diff
@@ -115,34 +117,38 @@ describe('HighlightListener', () => {
 
             highlightListener.handleMouseDown(new MouseEvent('mousedown', { buttons: 1 }));
 
-            expect(highlightListener.isMouseSelecting).toBe(true);
+            expect(setIsSelectingAction).toHaveBeenCalled();
+            expect(defaults.store.dispatch).toHaveBeenCalledWith(true);
```

Comment:
This is a little confusing, since dispatch should be called with an action, right?